theme_test_custo/data/images.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<record id="logo" model="theme.ir.attachment">
+    <field name="name">Test Theme Custom Logo</field>
+    <field name="url">/theme_test_custo/static/src/img/logo.jpg</field>
+    <field name="key">theme_test_custo.logo</field>
+</record>
+
+<!-- TODO: It is not yet possible to alter the website logo -->
+
+</odoo>
```

Comment:
I would remove that, can't be sure we'll allow to do that IMO. Except if that means less code to allow it but I don't think so.

Reply:
Sure :+1: 
We will be able to if that's part of the "style in db" task, isn't it?

Reply:
We could be able to with more code but I don't see the point in having a theme setting up a logo to be honest. If it was 0 line to allow it, sure. But I would not write one line of code to allow it TBH :shrug:

Reply:
I don't share the same opinion, as I have always said, the incoming success (or not) of Odoo or any such software is bound to its community themes capability.

I've always been convinced that if wordpress and bootstrap became what they became, its partly due to their template ecosystem, not the other way around.

That's just my deep belief of course.

And I have never seen a wordpress template or a bootstrap template without a custom logo :grimacing: 

AFAIC, this should 100% be possible and it is really important

Reply:
I don't see how the marketing of a theme is related to its use. You know I share your opinion about themes, having a logo of the theme set as default for your website has nothing to do with having a community that build themes. They are free to use a custom logo in their demo/marketing pages.

In the end, dumb debate, a community theme has already the possibility to use a custom header, they can add a custom editable image with a custom logo in there if they want (just like we do in our footers instead of using the website logo field). The website logo field will be removed at some point anyway I am sure (I am not sure it has any use case anymore actually? Except showing up in the website form view nobody ever see maybe?).

Reply:
EDIT 2: mmh ok, it has some uses as repeated between the navbar and the off canvas menu, also would create a cow'ed view if edited as a non-field image and probably would be more annoying for the configurator automatically setting up the logo

Reply:
Canceled two years ago: https://www.odoo.com/web#id=2212423&menu_id=4720&cids=1&action=4043&model=project.task&view_type=form but that can change :shrug: